File: Level_1/정수 제곱근 판별/Sqrt.java
class Solution {
    public long solution(long n) {
        long answer = 0;
        long q = (long)Math.sqrt(n);
        
        if((q*q) == n){
            answer = (long)Math.pow((q+1), 2);
        }else{
            answer = -1;
        }
        
        return answer;
    }
}

File: Level_1/정수 제곱근 판별/Sqrt.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
#include <math.h>

long long solution(long long n) {
    long long answer = 0;
    long s = (long)sqrt(n);
    if(s*s == n){
        answer = (long)pow((s+1), 2);
    }else{
        answer = -1;
    }
    return answer;
}
